docs/understanding-your-pipeline/data-products/index.md

```diff
@@ -72,6 +72,32 @@ Break down the barriers that exist between data producers and data consumers, by
 
 ![Key elements of a Data Product](images/key-elements-of-a-data-product.png)
 
+**Volume metrics**; data products can identify events ingested in your pipeline that match the configured event specifications. This allows your team to visualize the volume-related metrics for events being tracked to specific application IDs.
+
+**Benefits:**
+
+You will be able to view several items in the UI that help identify and detect anomalies or potential misconfigurations in trackers that are not sending the expected events. This is particularly useful during the development phase when implementing tracking for a specific application using [Snowtype](/docs/collecting-data/code-generation). These visual elements include:
+
+* A counter for each event specification, showing the total number of events detected from the tracked [applications IDs](/docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md#application-ids) in the last 30 days.
+* A 'last seen' field for each event specification, indicating when the last event matching the event specification was detected.
+* A list of application IDs from which events are being tracked, displayed for each event specification. For each application ID, a status will be shown with different colors:
+  * __Green__: Events conforming to the event specification are being tracked and identified with the specific application ID inherited from the configured [source applications](/docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md).
```

Comment:
`Events conforming to the event specification are being tracked` -> `Event specifications are being tracked`

Reply:
Yeap, wanted to be closer to that. 

Reading `Events conforming to the event specification are being tracked` makes me think that an event conforming to the Event Specification will be counted, which is not true, we require the event_specification context.